cardano-api/cardano-api.cabal

```diff
@@ -47,6 +47,7 @@ library
                         -- in this module
                         Cardano.Api.Orphans
                         Cardano.Api.SerialiseTextEnvelope
+                        Cardano.Api.InMode
```

Comment:
Why are you exposing this?

Reply:
@Jimbo4350, because of https://github.com/input-output-hk/cardano-node/pull/3824/files#diff-0f50be38be094020018b7669ec7f01ec93922b928eed2c5de6b8aa06654deb57R60

Reply:
..and generally https://github.com/input-output-hk/cardano-node/pull/3824/files#diff-0f50be38be094020018b7669ec7f01ec93922b928eed2c5de6b8aa06654deb57R192-R198